source/includes/mongodb-compatibility-table-ruby.rst

```diff
@@ -0,0 +1,203 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large no-padding
+
+   * - Ruby Driver
+     - MongoDB 4.4
+     - MongoDB 4.2
+     - MongoDB 4.0
+     - MongoDB 3.6
+     - MongoDB 3.4
+     - MongoDB 3.2
+     - MongoDB 3.0
+     - MongoDB 2.6
+     - MongoDB 2.4
+
+   * - 2.12
```

Comment:
Same comment as earlier in the PR. This is an _existing_ version and should not report compatibility with MongoDB 4.4.

Interesting to note that this includes the latest, released minor version, while the language compatibility table earlier in the PR did not.

@p-mongo @egiurleo: Please confirm that the language support tables for 2.12 and the previous versions below are correct.